jobs/README.md

```diff
@@ -4,6 +4,12 @@ In this section you can find some job openings listed by our community members.
 
 __Be advised that Job Openings older than 3 months will be removed for the sake of keeping current. Feel free to repost.__
 
+## Front End Developers @ [Ecoworld SA – Greekferries Club](https://greekferries-club.gr)
+Ecoworld SA is looking for 2 Front End Developers (Junior & Senior) for full-time, in-house employment
```

Comment:
> in-house

might be worth mentioning where this is (e.g. Heraklion, Athens, etc.)

Reply:
Good point. Added Heraklion on the title

Reply:
I forced pushed in order not to have multiple commits. The change I made is not reflected here as I can see...